src/db.py

```diff
@@ -31,9 +64,18 @@ def list_files(query: str, tags: list[str]) -> list[File]:
     ]
 
 
-def add_files(files: list[File]):
+def add_files(files: "list[File]"):
     """
     Add files to database.
     """
     # TODO: add to db
+    """
+    for file in files:
+        query = f"INSERT INTO files(name, drive_url) VALUES ('{file.name}', '{file.link}');"
+        cursor.execute(query)
+        db.commit()
+        print("added", file.name)
+
+    """
+    print("adding file")
```

Comment:
remove the changes in `db.py`, these should be in a separate pr

Reply:
removed